src/test/resources/json_examples/v7/supportMatrix.json.segments

```diff
@@ -0,0 +1,4 @@
+{"configs":{"
+MP33":{"supportedServers":["
+]},"MP32":{"supportedServers":[
```

Comment:
I am not sure this file. Do you need to just mention MP60?

Reply:
@rdebusscher we are stuck on this failure. Can you please shed some lights on this? Your help is very much appreciated.

Reply:
@Emily-Jiang What is the issue exactly? From the failed test run, it just seems a problem with some assertions on text values.
Not much time to go in detail for the moment.

Reply:
I found the error in the surefire-reports and changed the code to print the URL call which fails: Download failed. URI: http://127.0.0.1:9090/api/project?supportedServer=LIBERTY&selectedSpecs=CONFIG&selectedSpecs=FAULT_TOLERANCE&selectedSpecs=HEALTH_CHECKS&selectedSpecs=METRICS&selectedSpecs=OPEN_TRACING&selectedSpecs=OPEN_API&artifactId=liberty&buildTool=MAVEN expected:<200> but was:<400>

Reply:
I tried the URL locally running on liberty:
![image](https://github.com/eclipse/microprofile-starter/assets/1511708/13ad7136-58f6-40ab-8981-3da856cae90c)
So after I called: `http://127.0.0.1:9080/mp-starter/api/project?supportedServer=LIBERTY&selectedSpecs=CONFIG&selectedSpecs=FAULT_TOLERANCE&selectedSpecs=HEALTH_CHECKS&selectedSpecs=METRICS&selectedSpecs=OPEN_TRACING&selectedSpecs=OPEN_API&artifactId=liberty&buildTool=MAVEN`, I received back:```
 {
	"error": "One or more selectedSpecs is not available for the given mpVersion",
	"code": "ERROR003"
}
```

Reply:
I am not sure the version combination. 
What about the error for APITest?

Reply:
Which file needs to be updated? @rdebusscher

Reply:
Based on the log of the test

https://github.com/eclipse/microprofile-starter/pull/494/checks#step:7:17864

This is the failure
```
supportMatrix(org.eclipse.microprofile.starter.APITest)  Time elapsed: 0.319 sec  <<< FAILURE!
java.lang.AssertionError: Response of /7/supportMatrix 
```

And from the test method `APITest#supportMatrix`
````
    public void supportMatrix() throws FileNotFoundException {
        test(v7Matrix, "/7/supportMatrix");
````

we see that the contents of this file os compared to the response

````
v7Matrix = new File(getClass().getClassLoader().getResource("json_examples/v7/supportMatrix.json.segments").getFile());        
````